src/Behat/Gherkin/Parser.php

```diff
@@ -275,6 +275,26 @@ protected function parseFeature()
             }
         }
 
+        if ($background === null || !$background->hasExamples()) {
+            foreach ($scenarios as $scenario) {
+                if ($scenario instanceof OutlineNode && !$scenario->hasExamples()) {
+                    throw new ParserException(sprintf(
+                        'Outline should have examples table, but got none for outline "%s" on line: %d%s',
+                        rtrim($scenario->getTitle()),
+                        $scenario->getLine(),
+                        $this->file ? ' in file: ' . $this->file : ''
+                    ));
+                }
+            }
+        }
+        if ($background !== null && $background->hasExamples()) {;
```

Comment:
This could be `} else {` rather than the if statement that is exactly the opposite logic of line 278.